Sprint-1/destructuring/exercise-2/exercise.js

```diff
@@ -70,3 +70,51 @@ let hogwarts = [
     occupation: "Teacher",
   },
 ];
+
+// Task-1
+// Function to display names of people members of Gryffindor house
+
+function displayGryffindorPeople(hogwarts) {
```

Comment:
In my opinion 'hogwarts' as parameter name for this function can be renamed to something more appropriate. Yes, you will eventually pass 'hogwarts' to this method (on line 86). But this doesn't necessary mean that you need to create your parameter name with 'hogwarts'. Please, think of your method displayGryffindorPeople() as a standalone entity. In that case, ask your self what 'hogwarts' parameter will hold inside? It could include students from all houses, staff, and even other entities. In that case what will be a better name for this param?